packages/p2p/src/components/buy-sell/__tests__/buy-sell.spec.jsx

```diff
@@ -25,12 +24,24 @@ jest.mock('Stores', () => ({
     useStores: jest.fn(() => mock_store),
 }));
 
+jest.mock('@sendbird/chat', () => ({
+    SendbirdChat: jest.fn().mockReturnValue({}),
+}));
+
+jest.mock('@sendbird/chat/groupChannel', () => ({
+    SendbirdChat: jest.fn().mockReturnValue({}),
+}));
+
+jest.mock('@sendbird/chat/message', () => ({
+    SendbirdChat: jest.fn().mockReturnValue({}),
+}));
+
```

Comment:
This only checks the use case where SendBird is working as expected, we should handle the case where Sendbird might throw any error or fails : https://app.clickup.com/t/20696747/FEQ-209 FYI

Reply:
@amir-deriv, I have not written any test cases here. I've just added the `mocks`. The scope of the task is not to write tests. It's to fix then existing failing tests

Reply:
The test cases were written in [this](https://github.com/binary-com/deriv-app/pull/9457/files) PR

Reply:
Ok @jim-deriv , It''s fine. One thing I can see is we have a lot of duplicate mocking statement for this sendbird module we should mock it globally and change the mock as per our use case. but this PR is fine 👍